pkg/notify/discord.go

```diff
@@ -25,36 +25,39 @@ var embedColor = map[config.Level]int{
 // Discord contains URL and ClusterName
 type Discord struct {
 	log logrus.FieldLogger
+	api *discordgo.Session
 
 	Token     string
 	ChannelID string
 	NotifType config.NotifType
 }
 
 // NewDiscord returns new Discord object
-func NewDiscord(log logrus.FieldLogger, c config.Discord) *Discord {
+func NewDiscord(log logrus.FieldLogger, c config.Discord) (*Discord, error) {
+	api, err := discordgo.New("Bot " + c.Token)
```

Comment:
I'm not against that change, but why we need to do that? Is it safe to optimize here? (Personally, I didn't check how to use it and if it is thread safe). If you tested that, I'm fine with the change 👍

Reply:
Yep, we just use methods which do stateless HTTP requests underneath, so we don't need to create it every time to do the request. I tested that through and didn't notice any stability issues, so I'm keeping it 👍